sources/store.c

```diff
@@ -366,6 +366,7 @@ int CoSave(UBYTE *inp)
 	UBYTE *p, c;
 	WORD n = 0, i;
 	WORD error = 0, type, number;
+	WORD exprInStorage = 0;
```

Comment:
`exprInStorage` is used as its type is `BOOL` (whether any expressions are in the storage or not). I wonder using `WORD` for this variable might look like it represents "the number of expressions in the storage" at first glance. The variable name could be changed to one that clearly suggests a boolean value or the type could be changed to `BOOL` (though the `BOOL` is not used in traditional (old) code).

Reply:
Now I call it `exprInStorageFlag`, which should be more clear.